Python/Magic Pen/README.md

```diff
@@ -0,0 +1,4 @@
+# Magic-Pen
+Ever dreamed of drawing stuff on the screen just by waving your pen in the air...Well that is exactly what Magic Pen does using Python(OpenCV and NumPy).
+
```

Comment:
Add an instruction for getting started with the Script. It should look something like this: `python script.py`